src/watchful/client.py

```diff
@@ -1080,6 +1080,19 @@ def export_preview(mode: str = "ftc") -> Optional[Dict]:
 
     return api("export_preview", mode=mode)
 
+def export_project() -> http.client.HTTPResponse:
+    """
+    This function returns a consolidated version (a single *.hints file) of the
+    currently open project.
+    """
```

Comment:
To include the function return as well; the docstring will be useful for users and shown publicly, and we want to be consistent for all docstrings.

Reply:
included.